app/workers/education_form/send_sco_email.rb

```diff
@@ -0,0 +1,101 @@
+module EducationForm
+  class SendSCOEmail
+    include Sidekiq::Worker
+
+    def perform(user_uuid, claim_id)
+      @user = User.find(user_uuid)
+      @claim = SavedClaim::EducationBenefits::VA10203.find(claim_id)
+
+      email_sent(false)
+
+      @gi_bill_status = get_gi_bill_status
+      if less_than_six_months?
+        @facility_code = get_facility_code
+
+        if @facility_code.present?
+          @institution = get_institution
+
+          send_sco_email
+        end
+      end
+
+      @claim.save
+    end
+
+    private
+
+    def email_sent(sco_email_sent)
+      application = @claim.parsed_form
+      application['scoEmailSent'] = sco_email_sent
+      @claim.form = JSON.generate(application)
+    end
+
+    def get_gi_bill_status
+      service = EVSS::GiBillStatus::Service.new(@user)
+      service.get_gi_bill_status
+    rescue => e
+      Rails.logger.error "Failed to retrieve GiBillStatus data: #{e.message}"
+      {}
+    end
+
+    def get_facility_code
+      most_recent = @gi_bill_status.enrollments.max_by(&:begin_date)
+
+      return {} if most_recent.blank?
+
+      most_recent.facility_code
+    end
+
+    def get_institution
+      GIDSRedis.new.get_institution_details({ id: @facility_code })[:data][:attributes]
+    end
+
+    def less_than_six_months?
+      return false if @gi_bill_status.remaining_entitlement.blank?
+
+      months = @gi_bill_status.remaining_entitlement.months
+      days = @gi_bill_status.remaining_entitlement.days
+
+      ((months * 30) + days) <= 180
```

Comment:
Is this just a rough estimate? Not all months have 30 days 😄 .

Reply:
This is the same math as in frontend.
https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/edu-benefits/10203/prefill-transformer.js#L7
https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/edu-benefits/10203/containers/IntroductionPage.jsx#L17